api/default_networks.go

```diff
@@ -114,6 +116,52 @@ func optimismSepolia(stageName string) params.Network {
 	}
 }
 
+func base(stageName string) params.Network {
+	return params.Network{
+		ChainID:                BaseChainID,
+		ChainName:              "Base",
+		DefaultRPCURL:          fmt.Sprintf("https://%s.api.status.im/nodefleet/base/mainnet/", stageName),
+		DefaultFallbackURL:     fmt.Sprintf("https://%s.api.status.im/infura/base/mainnet/", stageName),
+		DefaultFallbackURL2:    fmt.Sprintf("https://%s.api.status.im/grove/base/mainnet/", stageName),
+		RPCURL:                 "https://base-mainnet.infura.io/v3/",
+		FallbackURL:            "https://base-archival.rpc.grove.city/v1/",
```

Comment:
there seems to be no `base-archival` endpoint in Grove, just `base`
https://docs.grove.city/base-api/intro